website/scripts/data/cli.json

```diff
@@ -214,6 +214,16 @@
           "spec": {
```

Comment:
Can we add something to the readme regarding this or will updating the website suffice ?

Also can we add an "inspiration" page to show that we looked into ahoy and choria-io to help build this interface ?

https://github.com/choria-io/appbuilder
https://github.com/ahoy-cli/ahoy